src/Grid/FeatureGrid/FeatureGrid.tsx

```diff
@@ -26,7 +32,22 @@ import {
   RgCommonGridProps
 } from '../commonGrid';
 
+interface HeaderCellProps extends React.HTMLAttributes<HTMLTableCellElement> {
+  id: string;
+}
+
+interface BodyCellProps extends React.HTMLAttributes<HTMLTableCellElement> {
+  id: string;
+}
+
+interface DragIndexState {
+  active: UniqueIdentifier;
+  over: UniqueIdentifier | undefined;
+  direction?: 'left' | 'right';
+}
+
 type OwnProps = {
+  draggableColumns?: boolean;
```

Comment:
Can we provide a short description for the api docs for the newly added property?